### DISCUSSION TITLE
Can't get Watch Folder to work

### DISCUSSION THREAD

**Discussion by ziaddemir** [CommandPost/CommandPost#3267] [Technical Support]
The workflow I had in mind is that I download images or videos into a folder and have them automatically added to Final Cut's media browser -- when I download new files however, I only get a notification saying "[file] is incoming..." that never resolves. I sort of? got it to work one time when I added a different Watch Folder and added new files to that: that time I did get a prompt to import the respective files, when I clicked it however I just got the spinning beach ball and then nothing. "Copy to library storage location" is toggled in Final Cut settings.

**Comment by latenitefilms** [comment]
I'll have to get back to you on this one. Watch Folders are only used by a small handful of users, and I'm pretty sure they're all running older versions of CommandPost. We haven't touched or updates the Watch Folder functionality is a looooooong time.

Do you get any errors in the Debug Console?

**Reply by ziaddemir** [reply]
Unfortunately no. Appreciate you taking the time to reply given this seems to be a pretty niche issue.
